projectEuler/problem3.js

```diff
@@ -1,7 +1,6 @@
 // prompt: what is the largest prime factor of the number 600851475143
 
-// isPrime checks to see if number argument is a prime integer by checking to see if the integer
-//  has more than one and itself as factors
+// isPrime checks to see if number argument is a prime integer by checking to see if the integer has more than one and itself as factors
```

Comment:
Your function name is self-documenting, no need for this comment. However if you wanted to keep this comment you could make it more concise it like so: `returns true if the integer has factors other than 1 and itself`